client/components/CardEntry.jsx

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
 
 // eslint-disable-next-line import/extensions
 import PhotosCarousel from './PhotosCarousel.jsx';
 
 const Card = styled.div`
-  background-color: rgb(250,250,250);
-  width: 720px;
-  height: 258px;
+  width: 50%;
+  min-width: 420px;
+  max-width: 897px;
+  max-height: 258px;
+  margin-bottom: 40px;
```

Comment:
I'm curious how you came up with these numbers? Inspecting the actual site? Very cool to make it more responsive!

Reply:
I literally measured the original site by resizing the screen to see how big or small the card would become. I already changed this tho by removing max-width and max-height since they were acting funny when I resized the window. My latest css is much leaner and responsive.